blog_tags.json

```diff
@@ -186,7 +186,7 @@
             "name": "OpenShift",
             "posts": ["graphql-open-liberty-20006","log4j-openshift-container-platform",
                       "infinispan-open-liberty-openshift", "openshift-oauth-server-social-login-liberty-20001",
-                      "ibm-red-hat-support-open-liberty", "prometheus-alertmanager-rhocp-open-liberty",
+                      "prometheus-alertmanager-rhocp-open-liberty",
                       "microprofile-3-3-open-liberty-20004"]
```

Comment:
can you combine the last two tags so they're on a single line (aka, two tags per line)